ui-tests/src/main/java/properties/PropertiesLoader.java

```diff
@@ -0,0 +1,59 @@
+package properties;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URISyntaxException;
+import java.util.Properties;
+
+/**
+ * Created by mastepan on 10/30/17.
+ */
+public final class PropertiesLoader {
+
+	private PropertiesLoader() {
+	}
+
+	private static Properties properties = new Properties();
```

Comment:
I'd like to see class like `TestConfiguration` that would define all property keys as constant and would expose methods to retrieve configuration without reusing keys.